### PR TITLE
Update modules for pipelines

### PR DESCRIPTION
**Pull request by sriharshakandala**
## Purpose 
Update modules for pipelines

----
- [x] I have read and checked the items on the review checklist.

### CODE REVIEW

**Comment by sriharshakandala** [comment]
https://buildkite.com/clima/rrtmgp-clima-a100-pipeline/builds/40#_

**Comment by Sbozzolo** [comment]
You can use julia 1.11 if you want, the build is green: https://github.com/CliMA/RRTMGP.jl/pull/553

**Comment by sriharshakandala** [comment]
> You can use julia 1.11 if you want, the build is green: #553
I would prefer to stay consistent with whatever `ClimaAtmos` is using for now! @charleskawczynski pointed out that there are some regressions with 1.11 that are still being worked on!

File: .buildkite/A100_pipeline/pipeline.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 agents:
   queue: clima
-  modules: climacommon/2024_04_05
+  modules: julia/1.10.5 cuda/julia-pref nsight-systems/2024.6.1
```
Comment (Sbozzolo) [inline .buildkite/A100_pipeline/pipeline.yml:3]:
```suggestion
  modules: climacommon/2024_10_08
```
(or 09 if you want julia 1.11)

Comment (Sbozzolo) [reply .buildkite/A100_pipeline/pipeline.yml:3]: You can also keep it explicit, except that `cuda/julia-pref` is not a valid module

Comment (sriharshakandala) [reply .buildkite/A100_pipeline/pipeline.yml]: Moved to `climacommon/2024_10_08`

**Comment by sriharshakandala** [comment]
https://buildkite.com/clima/rrtmgp-clima-a100-pipeline/builds/41#_

**Review by Sbozzolo — APPROVED** [review]

**Pull request merged**